src/main/java/com/example/spacecatmarket/javawebkpi/config/FeatureToggleProperties.java

```diff
@@ -0,0 +1,15 @@
+package com.example.spacecatmarket.javawebkpi.config;
+
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.Map;
+
+@Configuration
+@Data
```

Comment:
Bad practice, please never do this again. You configuration in the nutshell should be immutable